src/main/java/data/schedule/DayList.java

```diff
@@ -172,8 +172,12 @@ public void clearDayPlan(String userArgument) throws InvalidScheduleException {
             logger.log(Level.WARNING, "User entered an invalid day number.");
             throw new InvalidScheduleException(className, InvalidScheduleException.DAY_NUMBER_OUT_OF_RANGE);
         }
-        scheduleList[dayNumber - 1] = null;
-        dayList[dayNumber - 1] = null;
+        if (dayList[dayNumber - 1] == null) {
+            System.out.println("No plan is being scheduled, hence no plan is cleared.");
```

Comment:
Line 176: perhaps a suggestion: "No plan is scheduled hence there is nothing to clear"